module/userProjects/helper.js

```diff
@@ -4174,6 +4174,78 @@ module.exports = class UserProjectsHelper {
 			}
 		})
 	}
+
+	/**
+   * deleteUserPIIData function to delete users Data.
+   * @method
+   * @name deleteUserPIIData
+   * @param {userDeleteEvent} - userDeleteEvent message object 
+   * {
+      	"entity": "user",
+		"eventType": "delete",
+		"entityId": 101,
+		"changes": {},
+		"created_by": 4,
+		"organization_id": 22,
+		"tenant_code": "shikshagraha",
+		"status": "INACTIVE",
+		"deleted": true,
+		"id": 101,
+		"username" : "user_shqwq1ssddw"
+    }
+   * @returns {Promise} success Data.
+   */
+	static deleteUserPIIData(userDeleteEvent) {
+		return new Promise(async (resolve, reject) => {
+			try {
+				let userId = userDeleteEvent.id
+				if (!userId) {
+					throw {
+						status: HTTP_STATUS_CODE.bad_request.status,
+						message: CONSTANTS.apiResponses.USER_ID_MISSING,
+					}
+				}
+
+				let filter = {
+					userId: userId,
+				}
+
+				const updateProfile = {
+					$set: {
```

Comment:
@priyanka-TL the structure of user profile can change based on the user service we are using. please use some config map to determine the keys to update, And  the idea is that using this configuration we can manage it

Reply:
done